src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2718,7 +2815,29 @@ export class StudyViewPageStore
             this.updateClinicalDataIntervalFilters
         );
     }
-
+    @action.bound
+    setCustomChartNumericalFilters(
```

Comment:
i see below that we have a method
```setCustomChartFilters```
if this is in fact categorical, we should change the name to avoid confusion with the new numerical option

Reply:
I renamed `setCustomChartFilters` to `setCustomChartCategoricalFilters.` This filter takes input `values` (string[]) to filter categorical data. The `setCustomChartNumericalFilters` takes `dataBins` (DataBin[]) as input. It makes sense to have a more explicit name for the custom categorical data filter.